recipes/catch2/3.x.x/conanfile.py

```diff
@@ -23,12 +23,14 @@ class Catch2Conan(ConanFile):
         "fPIC": [True, False],
         "with_prefix": [True, False],
         "default_reporter": [None, "ANY"],
+        "console_width": range(80, 1000),
```

Comment:
years of doing conan and I did not know this was a thing https://github.com/conan-io/conan/issues/3357 🤯

Reply:
So I followed up with the Conan team and this is no longer the recommendation

Is should be an "any" value (see the one above) and if there's validation it should go in the `validate()` method

```suggestion
        "console_width": [None, "ANY"],
```